### PR TITLE
LPD-27268 Migrate some accessibility tests

### PR DESCRIPTION
**Pull request by veroglez**
In this PR, the checkAccessibility method has been implemented to test the accessibility of our components with axe-core in jest.

Axe in this method is configured to test under WCAGs 2.0A, 2.0AA, 2.1A, 2.1AA, 2.2AA and a `bestPractices` prop has been added (set to false by default) to add best practices if the test requires it.

At the moment the **color-contrast checks have been disabled since do not work in JSDOM** ([details about this](https://www.npmjs.com/package/axe-core/v/4.9.0-canary.56e139a#:~:text=There%20is%20limited%20support%20for%20JSDOM.%20We%20will%20attempt%20to%20make%20all%20rules%20compatible%20with%20JSDOM%20but%20where%20this%20is%20not%20possible%2C%20we%20recommend%20turning%20those%20rules%20off.%20Currently%20the%20color%2Dcontrast%20rule%20is%20known%20not%20to%20work%20with%20JSDOM)).

If there are no accessibility issues the tests will pass without problems. Otherwise it will show an array with all the issues found.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:playwright

File: modules/test/playwright/tests/site-admin-web/test.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+##
+## Testray
+##
+
+    testray.main.component.name=Site Admin
```
Comment (sandrodw3) [inline modules/test/playwright/tests/site-admin-web/test.properties:5]: This name should be the same we have [here](https://github.com/liferay/liferay-portal/blob/master/test.properties#L5276)

File: modules/test/playwright/tests/layout-admin-web/pageConfiguration.spec.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import {checkAccessibility} from '../../utils/checkAccessibility';
+
+const test = mergeTests(loginTest());
+
+test('checks the accessibility of the General page configuration', async ({
+	page,
+}) => {
+	await page.goto('/');
+
+	await page.getByLabel('Configure Page').click();
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-admin-web/pageConfiguration.spec.ts:18]: Maybe we should add a `waitFor` after this click to make sure we are already in the config view before checking accessibility

**Review by sandrodw3 — COMMENTED** [review]
This is awesome ❤️ love everything you are doing with axe utilities!

Just left some tiny comments. Also remember we have to add `site-admin-web` to the global `test.properties` in our Echo routine section, and also add a `test.properties` to the module itself in portal so playwright tests are ran on relevant when changing that module. Reach me in slack if you have doubts with this 🥰

File: modules/test/playwright/tests/layout-admin-web/pageConfiguration.spec.ts (hunk 2 above)
Comment (veroglez) [reply modules/test/playwright/tests/layout-admin-web/pageConfiguration.spec.ts:18]: The `waitFor` is done within the `checkAccessibility`, so we don't have to worry about it outside of this 🙌

**Comment by veroglez** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:forward

**Pull request closed without merging**